egs/chime1/s5/local/chime1_prepare_data.sh

```diff
@@ -41,7 +41,7 @@ rm -f "$scp"
 for sid in `seq 34`; do
   sid2=`printf "s%02d" $sid`
   ls -1 $wav_train/id$sid/*.wav \
-    | sed "s/\(.*\)\/\(.*\).wav/${sid2}_\2\t\1\/\2.wav/" \
+    | sed $"s/\(.*\)\/\(.*\).wav/${sid2}_\2\t\1\/\2.wav/" \
```

Comment:
this is not right: the dollar sign syntax doesn't work with double-quotes, and with single-quotes the backslash-then-digit things won't work right.  I suggest to do
` tab=$'\t'`
and then use `$tab` instead of `\t`.